app/controllers/submission-controller.js

```diff
@@ -47,11 +47,25 @@ router
  * Simply pipes well-formed request to the OpenRosa server and
  * copies the response received.
  *
- * @param {module:api-controller~ExpressRequest} req - HTTP request
- * @param {module:api-controller~ExpressResponse} res - HTTP response
+ * @param {express.Request} req - HTTP request
+ * @param {express.Response} res - HTTP response
  * @param {Function} next - Express callback
  */
 async function submit(req, res, next) {
+    if (!req.headers['content-type']?.startsWith('multipart/form-data')) {
+        res.status(400)
+            .set('content-type', 'text/xml')
+            .send(
+                /* xml */ `
+                <OpenRosaResponse xmlns="http://openrosa.org/http/response" items="0">
```

Comment:
It feels strange to me to see an `OpenRosaResponse` here but I do think it's the right thing. 

This comes back to the theme of "what's the separation of concerns". The Enketo Express backend has some concerns that do feel specific to Enketo and then others like this that feel like OpenRosa server concerns.

I would find it helpful to have documentation for the Enketo backend endpoints that are exposed for use by the frontend (as opposed to [the integration endpoints](https://apidocs.enketo.org/v2). We could make it explicit those are considered a "private" API and can change at any time and also do a careful review of each of them.

Reply:
I also felt it was weird to add this here, and considered just passing the request through to get the same response, but it seemed like an unnecessary network round trip for a known error condition.

Agree it would be good to explicitly document which APIs are private/internal.